jenkins/debian/Jenkinsfile_v4l2

```diff
@@ -0,0 +1,36 @@
+library identifier: 'jenkins-libs@master', retriever: modernSCM(
+  [$class: 'GitSCMSource',
+   remote: 'https://github.com/ana/kernelci-jenkins-libs'])
```

Comment:
Since the library is already pushed, i'll add a couple comments about it there.
* The labels are inverted with arm64 and armel : see https://github.com/ana/kernelci-jenkins-libs/blob/0d3e98c82509d2bd59cedbd581e16828da80a9c5/vars/buildTest.groovy#L16
* Couldn't some kind of struct be used rather than two associative arrays (debian_arch and toolchain_arch)?
* make sure to build the image once for each arch.
* I'll second what zumbi said about scripts file, this matches well the stage concept of jenkins